app.py

```diff
@@ -118,7 +119,7 @@ def index():
 def output():
     if flag:
         if os.environ['TRAVIS_TAG']:  # if TRAVIS_TAG have value it will proceed
-            os.system('./script.sh')
+            build.send_trigger_request(os.environ['email'], os.environ['TRAVIS_TAG'], os.environ['event_url'],os.environ['TRAVIS_SCRIPT'])
             print ('/build called')
```

Comment:
let's remove this print too this was there for debugging the Heroku logs very old and not useful fine

Reply:
@xeon-zolt could you also please describe the use of travis_script_1, we can use the travis_tag to the server in other ways